nodejs/db2util/index.js

```diff
@@ -0,0 +1,9 @@
+const callDb2util = require('./callDb2util');
+const config = require('./config')();
+
+config.query = 'SELECT * FROM QIWS.QCUSTCDT';
+
+callDb2util(config, (error, result) => {
+    if (error) { throw error; }
+    console.log(result);
```

Comment:
Should have a comment showing what the example output looks like. That way, people can easily grasp how to wire this into a larger application before even needing to deploy it themselves.